### ISSUE TITLE
Add a way to overview all starred community solutions under specific track

### ISSUE THREAD

**Issue by bogovich** [exercism/exercism#6306]
Hi,

It would be great to have a way to overview all starred community solutions under a specific track, as noted in the title of the issue.
I check out community solutions after every solved exercise to check for original and creative solutions to different exercises and try to understand them. 
But If I want to remind myself of these creative solutions I starred after a while, I first have to remember under which exercise did I star them, and then find them one by one. I think it would be a great improvement if I could see them all on one page so I can periodically consolidate my knowledge and understanding of them.

Thanks

**Comment by kytrinyx** [comment]
Just to clarify: you mean all the community solutions that you, specifically, have starred, right? Not all community solutions that have been starred by _someone_?

**Comment by bogovich** [comment]
Yes, that is what I meant, sorry for the lack of initial clarification. 
I only wish to overview the community solutions I have starred/saved, under a specific track. Just so I could have an additional focus on solutions that I found creative and interesting, without selecting each exercise individually. 
If this feature already exists, I apologize, I wasn't able to find it.

**Comment by kytrinyx** [comment]
Thanks for the clarification ✨

**Comment by ibafly** [comment]
I saw starring in exercism as a 'bookmark an interesting solution' thing until I couldn't find a place to view all my starred ones. My objective was to navigate to a specific code block in one solution but I can really not remember what exercise name it has.

**Comment by iHiD** [comment]
@ErikSchierboom This would be a valuable feature to add.

1. Should appear on the top-right drop down menu as "Starred Solutions"
2. A page that lists them with pagination. Ideally with track/exercise filter. You could probably copy/paste most of this from one of the other solution listing pages.
3. New API endpoint (probably at `/solutions/starred` as `starred` action on `api/solutions_controller`)
